frontend/src/components/InterviewPage.js

```diff
@@ -15,14 +15,33 @@ const InterviewPage = () => {
     const navigate = useNavigate()
 
     const interviewId = location.state.interviewId
+    const username = location.state.username
 
     useEffect(() => {
         MatchingService
             .getInterview(interviewId)
             .then(interviewDetails => {
-                console.log(interviewDetails)
                 setQuestionTitle(interviewDetails.data.question.title)
                 setQuestionDescription(interviewDetails.data.question.description)
+                const matchUsername = username === interviewDetails.data.firstUsername ? interviewDetails.data.secondUsername : interviewDetails.data.firstUsername
```

Comment:
What about `otherUsername` instead of `matchUsername` to show that it is the other party?

Reply:
In the table that shows the users history was `Matched Username`. I think `matchUsername` might be better as a variable? We can look into the variable naming next time if not the collections would have to be dropped to change the schema variable